tests/test_wrappers.py

```diff
@@ -538,6 +540,69 @@ def test_etag_response_mixin():
     response.make_conditional(env)
     assert response.content_length == 999
 
+    # simple valid Range Request
```

Comment:
I have no idea how we got into this situation, but make a separate test function for each test scenario even though the file indicates otherwise.